packages/ringcentral-integration/modules/Storage/index.js

```diff
@@ -81,7 +87,10 @@ export default class Storage extends StorageBase {
           type: this.actionTypes.resetSuccess,
         });
       }
-      if (this.status !== moduleStatuses.pending) {
+      if (
+        this.status !== moduleStatuses.pending &&
```

Comment:
I'd suggest to only sync on this.status === moduleStatuses.ready instead.
Even though it's equivalent to the current behavior, but I think explicitly use ready state is more straight forward.

Reply:
OK, I also feel so good.